static/app/components/replays/shareButton.tsx

```diff
@@ -17,12 +17,15 @@ import {useRoutes} from 'sentry/utils/useRoutes';
 function ShareModal({currentTimeSec, Header, Body}) {
   const routes = useRoutes();
   const [isCustom, setIsCustom] = useState(false);
+  const [isShareWithTimestamp, setIsShareWithTimestamp] = useState(true);
```

Comment:
woah, if we're not including the `?t=#` param in the url, then there's no point in the modal. -> the url without the `t` param is the same url that's in the browser, you could just copy the url from there


The point of the modal is to add the `?t=` param, and the time is either going to be the current player time, or some custom timestamp you can type in (and we'll translate the hh:mm format into number of seconds).

I'm going to comment in the original bug ticket, since there is confusion the right fix might be to improve the design of the modal instead.

Reply:
@ryan953 should be good to review again!